deps/rabbit/src/rabbit_queue_member_eval.erl

```diff
@@ -0,0 +1,261 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% Copyright (c) 2007-2023 VMware, Inc. or its affiliates.  All rights reserved.
+%%
+
+-module(rabbit_queue_member_eval).
```

Comment:
I think this module name is confusing. It is likely to make user think of `rabbitmqctl eval` or something similar, whereas what it actually does is continuous QQ member reconciliation (or repair).

I'd use a name like `rabbit_quorum_queue_periodic_membership_reconciliation`. On this team we do not mind longer names that are more to the point.

Reply:
OK, very good to know!